crates/bitwarden-crypto/src/safe/password_protected_key_envelope.rs

```diff
@@ -447,6 +442,29 @@ impl From<TryFromIntError> for PasswordProtectedKeyEnvelopeError {
     }
 }
 
+#[cfg(feature = "wasm")]
+#[wasm_bindgen::prelude::wasm_bindgen(typescript_custom_section)]
+const TS_CUSTOM_TYPES: &'static str = r#"
+export type PasswordProtectedKeyEnvelope = Tagged<string, "PasswordProtectedKeyEnvelope">;
+"#;
+
+#[cfg(feature = "wasm")]
+impl wasm_bindgen::describe::WasmDescribe for PasswordProtectedKeyEnvelope {
+    fn describe() {
+        <String as wasm_bindgen::describe::WasmDescribe>::describe();
+    }
+}
+
+#[cfg(feature = "wasm")]
+impl FromWasmAbi for PasswordProtectedKeyEnvelope {
```

Comment:
concern `FromWasmAbi` is marked as unstable. https://docs.rs/wasm-bindgen/0.2.104/wasm_bindgen/convert/trait.FromWasmAbi.html#-unstable

Reply:
Yea it's unstable but a lot of things depends on it and we'll know if it stops compiling.